### PR TITLE
Clean up the standalone examples

### PR DESCRIPTION
**Pull request by abarth**
Our examples have been growing organically over time. This patch cleans
them up to illustrate specific aspects of Flutter.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie @chinmaygarde 

I'll add more readmes and comments in future patches.  This patch was already getting kind of out of control.

I haven't fully cleaned out the old widgets examples yet.  That'll have to wait for a future patch too.

File: examples/layers/raw/hello_world.dart [hunk 1]
```diff
@@ -26,7 +25,11 @@ ui.Picture paint(ui.Rect paintBounds) {
 
 ui.Scene composite(ui.Picture picture, ui.Rect paintBounds) {
   final double devicePixelRatio = ui.window.devicePixelRatio;
-  ui.Rect sceneBounds = new ui.Rect.fromLTWH(0.0, 0.0, ui.window.size.width * devicePixelRatio, ui.window.size.height * devicePixelRatio);
+  ui.Rect sceneBounds = new ui.Rect.fromLTWH(
+      0.0,
+      0.0,
+      ui.window.size.width * devicePixelRatio,
+      ui.window.size.height * devicePixelRatio);
```
Comment (Hixie) [inline examples/layers/raw/hello_world.dart]: newline before )

File: examples/layers/raw/hello_world.dart [hunk 2]
```diff
@@ -52,8 +55,7 @@ void handlePopRoute() {
 
 void handlePointerPacket(ByteData serializedPacket) {
   bindings.Message message = new bindings.Message(
-      serializedPacket, <core.MojoHandle>[],
-      serializedPacket.lengthInBytes, 0);
+      serializedPacket, <core.MojoHandle>[], serializedPacket.lengthInBytes, 0);
```
Comment (Hixie) [inline examples/layers/raw/hello_world.dart]: ditto

**Comment by Hixie** [comment]
lgtm. it's hard to see some of these examples go, but it is time.

**Comment by Hixie** [comment]
side note: update-packages actually failed, but travis didn't notice.
it'll get fixed when you finish cleaning out the old examples/widgets directory

**Comment by abarth** [comment]
Will fix shortly.

**Pull request merged**